### ISSUE TITLE
The a9 core is not interpreted correctly by HFM

### ISSUE THREAD

**Issue by rhavern** [harlam357/hfm-net#363]
The new a9 core is not interpreted correctly by HFM.  The a9 core can run as CPU or GPU, depending on how the researcher deploys the project.  I'll attach log files for CUDA, OpenCL and CPU WUs.
[20220908 HFM a9 CUDA GPU log.txt](https://github.com/harlam357/hfm-net/files/9528717/20220908.HFM.a9.CUDA.GPU.log.txt)
[20220908 HFM a9 OpenCL GPU log.txt](https://github.com/harlam357/hfm-net/files/9528718/20220908.HFM.a9.OpenCL.GPU.log.txt)
[20220908 HFM a9 CPU log.txt](https://github.com/harlam357/hfm-net/files/9533498/20220908.HFM.a9.CPU.log.txt)

**Comment by harlam357** [comment]
Yes, you're right. A9 is mapped to CPU. However, that mapping should only apply queries of the WU database. It does not save the slot type and maps the core name / id to a slot type. Is that what you're seeing @rhavern?
